interface/forms/dictation/new.php

```diff
@@ -1,42 +1,73 @@
 <?php
 /**
- * Generated DocBlock
- *
- * @package OpenEMR
- * @link    http://www.open-emr.org
- * @author  cfapress <cfapress>
- * @author  bradymiller <bradymiller@users.sourceforge.net>
- * @author  Robert Down <robertdown@live.com>
- * @author  Brady Miller <brady.g.miller@gmail.com>
- * @author  Brady Miller <brady.g.miller@gmail.com>
- * @copyright Copyright (c) 2008 cfapress <cfapress>
- * @copyright Copyright (c) 2013 bradymiller <bradymiller@users.sourceforge.net>
- * @copyright Copyright (c) 2017 Robert Down <robertdown@live.com>
- * @copyright Copyright (c) 2017 Brady Miller <brady.g.miller@gmail.com>
- * @copyright Copyright (c) 2016 Brady Miller <brady.g.miller@gmail.com>
- * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
- */
+- * Generated DocBlock
+ - *
+ - * @package OpenEMR
+ - * @link    http://www.open-emr.org
+ - * @author  cfapress <cfapress>
+ - * @author  bradymiller <bradymiller@users.sourceforge.net>
+ - * @author  Robert Down <robertdown@live.com>
+ - * @author  Brady Miller <brady.g.miller@gmail.com>
+ - * @author  Brady Miller <brady.g.miller@gmail.com>
+ - * @copyright Copyright (c) 2008 cfapress <cfapress>
+ - * @copyright Copyright (c) 2013 bradymiller <bradymiller@users.sourceforge.net>
+ - * @copyright Copyright (c) 2017 Robert Down <robertdown@live.com>
+ - * @copyright Copyright (c) 2017 Brady Miller <brady.g.miller@gmail.com>
+ - * @copyright Copyright (c) 2016 Brady Miller <brady.g.miller@gmail.com>
```

Comment:
@robertdown ,
only 3 :)
I think this was from the auto generated docblocks(looks like used the github history to produce them) for the PSR2 megacommit to ensure headers at top of all php scripts.